src/main/java/io/github/wimdeblauwe/hsbt/thymeleaf/HtmxAttributeProcessor.java

```diff
@@ -14,22 +17,25 @@
 import org.thymeleaf.util.Validate;
 import org.unbescape.html.HtmlEscape;
 
+import java.util.LinkedHashMap;
+
 public class HtmxAttributeProcessor extends AbstractStandardExpressionAttributeTagProcessor
         implements IAttributeDefinitionsAware {
 
-
     public static final int ATTR_PRECEDENCE = 1000;
     private final String attrName;
+    private final ObjectMapper mapper;
 
     private static final TemplateMode TEMPLATE_MODE = TemplateMode.HTML;
 
     private AttributeDefinition targetAttributeDefinition;
 
 
     public HtmxAttributeProcessor(final String dialectPrefix,
```

Comment:
Maybe to be consistent, we can put each parameter on a new line. I would also remove the `final` of `dialectPrefix`, I find this is noise for little gain.